core/neat/mixins/_grid-media.scss

```diff
@@ -46,22 +57,32 @@
 ///     margin-left: 20px;
 ///   }
 ///
-///   @media only screen and (min-width: 1000px) {
+///   @media only screen and (max-width: 999px) {
 ///     .element {
 ///       width: calc(50% - 75px);
 ///       float: left;
 ///       margin-left: 50px;
 ///     }
 ///   }
+///
+///   @media only screen and (min-width: 1000px) {
+///     .element {
+///       width: calc(50% - 75px);
+///       float: left;
+///       margin-left: 70px;
+///     }
+///   }
 
-@mixin grid-media($grid) {
-  $_media: _retrieve-neat-setting($grid, media);
-  $_query: _neat-parse-media($_media);
+@mixin grid-media($grid...) {
```

Comment:
Even though this argument name is the same, it seems that changing it to an arglist still creates a breaking change. I threw together [a simple CodePen](https://codepen.io/tysongach/pen/prNpQM?editors=0100) and if I call the mixin (as I would in its current 2.1 version) using a named argument, swapping the mixin to the arglist throws an error:

> Argument $grid of mixin grid-media cannot be used as a named argument.

If that’s the case, I don’t believe we’d be able to ship in a minor release; would need a major release up to 3.0.